extensions/amp-script/amp-script.md

```diff
@@ -138,6 +138,51 @@ Will be reflected on the page as a new child of the `amp-script` element:
 
 Under the hood, `amp-script` uses [@ampproject/worker-dom](https://github.com/ampproject/worker-dom/). For design details, see the ["Intent to Implement" issue](https://github.com/ampproject/amphtml/issues/13471).
 
+### Advanced usages
+
+#### Passing data between amp-script and the page
+
+`amp-script` supports getting and setting [`amp-state`](https://amp.dev/documentation/components/amp-bind/#initializing-state-with-amp-state) JSON via JavaScript.
+
+This enables advanced interactions between `amp-script` and other AMP elements on the page via `amp-bind` [bindings](https://amp.dev/documentation/components/amp-bind/#bindings).
+
+[tip type="default"]
+`AMP.setState()` requires the [`amp-bind`](https://amp.dev/documentation/components/amp-bind) extension to be installed.
```

Comment:
Is `amp-bind` installed differently when used with `amp-script`? Is `installing` a component different than just including the script tag in the head?

Reply:
It's just including the script tag. Reworded this.

Reply:
Much better, thanks! :)